components/Common/CrossLink/index.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from './index.module.scss';
+import Link from 'next/link';
+import classNames from 'classnames';
+import { ChevronLeftIcon, ChevronRightIcon } from '@heroicons/react/24/solid';
+import { FormattedMessage } from 'react-intl';
+import type { FC } from 'react';
+
+type CrossLinkProps = {
+  direction: 'left' | 'right';
+  text: string;
+  url: string;
+};
+
+const CrossLink: FC<CrossLinkProps> = ({ direction, text, url }) => {
```

Comment:
I am aware that there's a smarter way of writing this by using `flex-row-reverse` and `rotate-{angle}`, see [this](https://github.com/reactjs/react.dev/blob/main/src/components/DocsFooter.tsx#L59).

However, I think it's better to keep it simple to make it easier to read.

Reply:
The way you did it seems better to me 😄

Reply:
I think it might make sense to use such a plugin instead of forcing certain themes for stories, but I'm not sure if it can be compared to a dark / white theme on the Chromatic side.
Something has been shared recently, it can be examined and checked for usability; https://www.chromatic.com/blog/introducing-story-modes/

Reply:
Thanks! I just skimmed through it and I think it's similar to https://github.com/nodejs/nodejs.org/pull/5781#discussion_r1321595086. I'll give it an eye later.

Reply:
I like this more complex approach. You can always simple document the references.

Reply:
Aesthetically, I also prefer the more complex approach because there are less `if-else`s.

Initially I thought it will be harder to read by newer/beginner devs. After re-considering, maybe it's not hard after all if we clarify the intention in CSS selector name.

I'll change to the more complex approach.